recipes/allennlp-server/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: allennlp-server
+  version: {{ version }}
+
+source:
+  url: https://github.com/allenai/allennlp-server/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 2cb54ae493921a0225e780fb045c62de6f9b52b4c2f39cae51c01bc2def040d7
+
+build:
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # currently missing windows / osx builds for allennlp >= 2.0
```

Comment:
If this is just dealing with missing dependencies on macos/windows, you can remove the `skip: true` and instead add `noarch: python`. We'll just ignore the failing builds since `noarch` recipes are only built on linux.

Reply:
Hey, thanks for the review!

I know that that would be possible, but there have been failures where things should have been noarch with other allennlp-packages, and I don't want this to get installed without testing, as soon as (sometime down the road) allennlp 2.0 is available on win/osx.

I can change it of course, if you feel strongly about that.

Reply:
AllenNLP is already available on OS X. It's fully supported. Everything should work. I'm not aware of any platform specific code in the library that would prevent that.

Reply:
@dirkgr, conda-forge is not building for osx yet due to missing (recent) pytorch builds. That in itself is not an argument against noarch, but there have been unexplained segfaults with python 3.9 for example, and I would just rather have the assurance of running the CI in all variants successfully for a while before I switch to noarch.

Reply:
Can you add your last comment as a comment in the recipe?